### PR TITLE
Create  UI console

### PR DESCRIPTION
**Pull request by SebastianSolisFenger**
## PR changes

- Add UI for all the features of the app
- Enable the user to add person, book and rentals
- Create app starter file and starter function
- Create an App class to hold the logic of the application
- Use exit to exit out of the application

## SCREENSHOTS

image
image

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -1,10 +1,157 @@
...
+  def rental_by_id
+    @people.each_with_index.map do |person, i|
+      puts "#{i}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, age: #{person.age}"
+    end
+    print 'Please, write the ID of the person: '
+    person_id = gets.chomp.to_i
+    puts 'Rentals: '
+    @my_rentals.each do |rental|
+      if rental.person.id == person_id
+        puts "Date: #{rental.date}, Book: #{rental.book.title} by Author: #{rental.book.author} "
```
Comment (FrederickMih) [inline app.rb]: - [ ] Hi @SebastianSolisFenger, you have built a fantastic app :+1: . This app is working as expected but there's a minute issue that you need to look into once more and your app will be approved immediately. The app could list a rental for a particular person by id but unfortunately, it got exited immediately on its own. I don't think this is normal. Kindly look into this minute issue. The only option to exit the app should be the number **"7 - Exit"**
- `Screenshot:`


Screenshot from 2022-07-20 21-39-49

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @SebastianSolisFenger  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




# To Highlights

- GitHub workflow has been followed :heavy_check_mark: 
- PR has a title and summary :heavy_check_mark: 
- Linter checks are Ok :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (SebastianSolisFenger) [reply app.rb]: hi @FrederickMih! Ok, it's working now.

image

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @SebastianSolisFenger 

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**